components/clarinet-format/tests/golden-intended/clarity-bitcoin.clar

```diff
@@ -264,7 +264,8 @@
         index: uint,
       },
       remaining: uint,
-      txins: (list 8 {
+      txins: (list 8
+        {
```

Comment:
not sure I follow this change.
If this `}` is on a new line, I think the indentation level shold be increased for the next lines.
And the matching `}` should be on the same indentation level

```clarity
      txins: (list 8
        {
          outpoint: {
            hash: (buff 32),
            index: uint,
          },
          scriptSig: (buff 256), ;; just big enough to hold a 2-of-3 multisig script
          sequence: uint,
        }
      ),
```

Reply:
Good catch. I believe this is because there's a difference between checking spans (list breaking on max lines) and the original place the opening bracket would have been 
```
(list 8 {
  ...
})
```

So, the list gets broken up onto multiple lines and the closing parens are still under the impression the `(` and `{` were opened on the same line.. Tricky, but i've made an issue [here](https://github.com/hirosystems/clarinet/issues/1810)